### DISCUSSION TITLE
media buttons not working

### DISCUSSION THREAD

**Discussion by PMortis2** [KRTirtho/spotube#204] [Q&A]
play previous, pause and play next buttons doesn't works, it's my problem or it's gonna be added in a further version? im really used to use my mouse buttons as media buttons and it triggers me a lot i can't use them, i hope it is gonna get added soon if it's not something happening to me

**Comment by KRTirtho** [comment]
Are you using Windows, 'cause Media Buttons aren't supported on Windows

**Reply by PMortis2** [reply]
yeah i do, it is just not supported atm, or there's something that doesn't allow to implement it for the windows client?
